packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/send/dq-requests.ts

```diff
@@ -47,27 +45,15 @@ export async function sendSignedDQRequests({
       };
       //eslint-disable-next-line @typescript-eslint/no-explicit-any
     } catch (error: any) {
-      const msg = "HTTP/SSL Failure Sending Signed DQ SAML Request";
+      const msg = "Failure Sending SAML Request";
```

Comment:
What do you think we always use `Error` when referring to these situations? If we use `Failure` as well, it makes the filter for logs harder - also when we're debugging manually on the logs.